### PR TITLE
[Website] Impact Numbers

### PR DESCRIPTION
**Pull request by FaithDaka**
#### Summary of Changes (What does this PR do?)

- Features impact numbers on homepage

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] The title of the PR states what changed and the related issues number (used for the release note).

#### How should this be manually tested?

- View impact section on homepage in deploy preview

#### What are the relevant tickets?

- [WEB-354]()

#### Screenshots 
image

image

[WEB-354]: https://airqoteam.atlassian.net/browse/WEB-354?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: website/frontend/src/pages/OurSolutions/ResearchPage.js
Comment (Baalmart) [inline website/frontend/src/pages/OurSolutions/ResearchPage.js:121]: I can seen some content changes, perhaps we need to also include this in the PR description. Thanks.

**Review by Baalmart — APPROVED** [review]
Thanks @FaithDaka for this. All LGTM to me.
I will wait for more reviews from our Designers , @Codebmk and @123MwanjeMike

**Review by 123MwanjeMike — APPROVED** [review]
Thanks @FaithDaka 
LGTM

**Pull request merged**
